internal/cli/atlas/quickstart/quick_start.go

```diff
@@ -292,6 +345,39 @@ func (opts *Opts) validateUniqueUsername(val interface{}) error {
 	return fmt.Errorf("a user with this username %s already exists", username)
 }
 
+func askOpenBrowser() (bool, error) {
```

Comment:
[nit] naming, `openMogoshDownloadPage`? something more clear as you would not know what page this is going to take you just on the method name